x-pack/plugins/profiling/server/lib/setup/steps/component_template_profiling_executables.json

```
@@ -0,0 +1,30 @@
+{
+  "settings": {
+    "index": {
+      "refresh_interval": "10s"
+    }
+  },
+  "mappings": {
+    "_source": {
+      "mode": "synthetic"
+    },
+    "properties": {
+      "ecs.version": {
+        "type": "keyword",
+        "index": true
+      },
+      "Executable.build.id": {
+        "type": "keyword",
+        "index": true
+      },
+      "Executable.file.name": {
+        "type": "keyword",
+        "index": true
+      },
+      "@timestamp": {
+        "type": "date",
+        "format": "epoch_second"
+      }
```

```suggestion
      },
      "Symbolization.lastprocessed": {
        "type": "date",
        "format": "epoch_second",
        "index": false
      }
```
This has been added in prodfiler recently.